src/main/java/com/easypost/easyvcr/internalutilities/ConsoleFallbackLogger.java

```diff
@@ -0,0 +1,53 @@
+package com.easypost.easyvcr.internalutilities;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * A logger that, if there's no proper logger, writes to the console instead.
```

Comment:
Nit: I'd clarify this maybe? If there is no `custom` or `overridden` logger, right?